tests/lang/tests/var/var_err.yaiwr

```diff
@@ -0,0 +1,6 @@
+// Run-time:
+//    stdout:
+//    stderr: Evaluation error: Cannot reference undeclared variable by 'a' id!
```

Comment:
I find this error message a bit "implementation driven".

Wouldn't a better error for the user be something like:

> Use of undefined variable `a`

?

Reply:
oki. changed :point_right: 9978f89a2ca8cf08f89e2cb45226d9e4e7370b63